assets/css/lesson.scss

```diff
@@ -321,6 +321,9 @@ span.fold-unfold {
   opacity: 0.5;
 }
 
+img {
```

Comment:
```suggestion
/*
  Force a pixelated rendering of images when original resolution is bigger than
  the <img> viewport. This issue affects primarily Chrome based browsers (as of 2021).
  See https://github.com/carpentries/styles/pull/636 for the original report and related links 
*/
img {
```

Reply:
@maxim-belkin does this look reasonable?

Reply:
> @maxim-belkin does this look reasonable?

Yes, thank you!